scripts/sql/57_update_table.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "public"."installed_apps" ADD COLUMN "deployment_app_type" varchar(50);
+
+update installed_apps set deployment_app_type='helm' WHERE app_id in (SELECT id from app WHERE app_offering_mode='EA_ONLY');
+
+update installed_apps set deployment_app_type='argo_cd' WHERE app_id in (SELECT id from app WHERE app_offering_mode='FULL');
```

Comment:
can you check if app_offering_mode is blank for some rows?

Reply:
!= EA_ONLY

Reply:
done